bin/install

```diff
@@ -25,4 +25,9 @@ sudo chmod +x ${EXECUTABLE}
 echo ">>> COPYING PLIST TO INSTALL DIRECTORY"
 cp ${PLIST} ${INSTALL_DIR}
 
+
 echo ">>> INSTALLED"
+echo "alias ra=\"VERSION=\"2.0.0\" ra --home \$HOME\"" >> ~/.bashrc
```

Comment:
You can remove the `--home` flag and `$HOME` value now; this value is pulled in from the OS in Java. Later on, it'd be nice to figure out a way to get rid of the version variable too.